lib/lazy_html/tree.ex

```diff
@@ -172,6 +172,34 @@ defmodule LazyHTML.Tree do
   @doc """
   Performs a depth-first, post-order traversal of the given tree.
 
+  This function traverses the tree without modifying it, check `postwalk/2` and
+  `postwalk/3` if you need to modify the tree.
+  """
+  @spec postreduce(
+          t(),
+          acc,
+          (html_node(), acc -> acc)
+        ) :: acc
+        when acc: term()
+  def postreduce([], acc, _fun), do: acc
```

Comment:
We can add body-less clause, so the signature shows nicely.

```suggestion
  def postreduce(tree, acc, fun)

  def postreduce([], acc, _fun), do: acc
```